data-es6.js

```diff
@@ -13910,6 +13931,7 @@ exports.tests = [
       */},
       res: {
         xs6:         false,
+        chrome50:    true,
```

Comment:
Yes, and this "fix" actually is backported to 49. Unfortunately, I'm not confident that the web can support as strict RegExp semantics as TC39 called for. My plan is to collect data and revert the workaround once I have more confidence in the web compatibility of these changes.